sys/cam/scsi/scsi_cd.c

```diff
@@ -2024,8 +2024,7 @@ cdioctl(struct disk *dp, u_long cmd, void *addr, int flag, struct thread *td)
 				data->header.data_len[1] +
 				sizeof(struct cd_sub_channel_header)));
 			cam_periph_unlock(periph);
-			error = copyout(data, __USER_CAP_UNBOUND(args->data),
```

Comment:
I think I found this because there's a translation for the ioctl in the Linuxulator code with the following scary comment:
```
/*
 * Invoke the native ioctl and bounce the returned data through
 * the userspace buffer.  This works because the Linux structure
 * is the same size as our structures for the subchannel header
 * and position data.
 */
```
and I wanted to be sure this was still the case for CheriABI. Perhaps there should be a static assertion for that property...

Reply:
Ugh, these all need to die. We managed to kill the ones that translated network related ioctl in completely unsafe ways.